src/abstractdataframe/abstractdataframe.jl

```diff
@@ -2239,3 +2239,38 @@ Base.getindex(::AbstractDataFrame, ::Union{Symbol, Integer, AbstractString}) =
 
 Base.setindex!(::AbstractDataFrame, ::Any, ::Union{Symbol, Integer, AbstractString}) =
     throw(ArgumentError("syntax df[column] is not supported use df[!, column] instead"))
+
+"""
+    reverse(df::AbstractDataFrame)
+
+This function reverses the current order of the rows in `df`.
+
+# Examples
+
+```jldoctest
+julia> df = DataFrame(a = 1:5, b = 6:10, c = 11:15)
```

Comment:
can you please remove spaces everywhere in this PR around `=` as kwarg? This is the style in Julia codes that you should not have spaces here. I have made one suggestion, but in other places the same applies.

Reply:
Sure... I was confused by looking at other places, so used spaces around `=`. Now I'll remove those.

Reply:
Can you x-link me a place in DataFrames.jl repo where we have spaces around `=`. We should remove them. Thank you!

Reply:
Umm.. Ok but, what's x-link btw?

Reply:
cross link - i.e. in which line of which file you have spotted the inconsistency. Thank you

Reply:
There are quite a few places to mention:
https://github.com/JuliaData/DataFrames.jl/blob/73f11b6a0558cca2847b79223b84ab35526d5be6/src/abstractdataframe/abstractdataframe.jl#L158
https://github.com/JuliaData/DataFrames.jl/blob/73f11b6a0558cca2847b79223b84ab35526d5be6/test/dataframe.jl#L660-L682
https://github.com/JuliaData/DataFrames.jl/blob/73f11b6a0558cca2847b79223b84ab35526d5be6/src/abstractdataframe/abstractdataframe.jl#L279

These are few places I've observed. May be I'll remove those spaces around `=` for kwargs in this PR itself.

Reply:
Better do this separately I'd say, to keep commits clean.

Reply:
@Chandu-4444 - I agree with @nalimilan in general (and please use separate PRs for clean up tasks).

@nalimilan - however, since @Chandu-4444 already did the job, I am OK to accept it in this PR.